sdk/storage/Azure.Storage.Blobs/src/Models/BlobProperties.cs

```diff
@@ -0,0 +1,261 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Storage.Blobs.Models;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Propeties of a Blob
+    /// </summary>
+    public partial class BlobProperties
```

Comment:
We didn't need the ```BlobPropertiesInternal _properties``` property, since were using ```BlobExtensions``` to do our conversion from ```BlobPropertiesInternal``` to ```BlobProperties```.